docs/docs/concepts/blueprints.md

```diff
@@ -1,61 +1,153 @@
 ---
-title: Blueprints
 sidebar_position: 1
 ---
 
-The data modelling tool is a tool that can be used for modelling complex domain models.
+# Blueprints and entities
 
-## What is a Blueprint?
+Blueprints can be used to create domain models. 
+
+:::info
+
+A blueprint is a model and an instance of a blueprint is called an entity. 
 
-A domain model is called for blueprint.
+:::
 
-An instances of a blueprint is called entitiy.
+## What is a Blueprint?
+
+A blueprint is described in [JSON](https://www.json.org/json-en.html). 
 
-Below is a short version of a blueprint for a Car.
+It is an JSON object that maps keys to values.
 
+Below is a short version of a Car blueprint. 
 
 ```json
 {
-  "type": "system/SIMOS/Blueprint",
   "name": "Car",
+  "type": "dmss://system/SIMOS/Blueprint",
   "description": "This describes how a car looks like",
   "attributes": [
     {
       "attributeType": "string",
-      "type": "system/SIMOS/BlueprintAttribute",
+      "type": "dmss://system/SIMOS/BlueprintAttribute",
       "name": "name"
     },
     {
       "attributeType": "string",
-      "type": "system/SIMOS/BlueprintAttribute",
+      "type": "dmss://system/SIMOS/BlueprintAttribute",
       "name": "description"
     }
+  ]
 }
 ```
+Any blueprint can be understood by simply looking at it. 
+
+In the example above, you can find out, by looking at its attributes: `name`, `type` and `attributes`. 
+
+* That it represent a Car model 
+* It is of type `system/SIMOS/Blueprint`
+* It has the two attributes `name` and `description`.
+
+## How to create a Blueprint?
+
+The `name` should reflects what the model is.
```

Comment:
```suggestion
The `name` should reflect what the model is.
```
or remove it entirely

Reply:
I have updated the text with the suggested changes.